src/shared/ui/LongTextInput.tsx

```diff
@@ -0,0 +1,52 @@
+import { TextInput } from 'react-native';
+
+import { GetProps, setupReactNative, styled } from '@tamagui/core';
+import { focusableInputHOC } from '@tamagui/focusable';
+
+setupReactNative({
+  TextInput,
+});
+
+export const LongTextInput = styled(
+  TextInput,
+  {
+    name: 'ParagraphText',
+
+    m: 0,
+    px: 12,
+    py: 8,
+
+    minWidth: 0,
+    minHeight: 56,
+    maxHeight: 350,
+    maxWidth: '100%',
+    width: '100%',
+    borderRadius: 4,
+    borderWidth: 2,
+    outlineWidth: 0,
+    padding: 8,
```

Comment:
`padding` is unused in favor for `px, py`

Reply:
I removed the px and py and followed the figma style in my next commit.